ng-dev/pr/merge/defaults/labels.ts

```diff
@@ -32,15 +33,25 @@ import {assertActiveLtsBranch} from './lts-branch';
  * @param releaseConfig Configuration for the release packages. Used to fetch
  *   NPM version data when LTS version branches are validated.
  */
-export async function getDefaultTargetLabelConfiguration(
-  api: GithubClient,
-  githubConfig: GitClientConfig,
-  releaseConfig: ReleaseConfig,
+export async function getTargetLabels(): Promise<TargetLabel[]>;
+export async function getTargetLabels(
+  _api?: GithubClient,
+  _config?: Partial<{github: GitClientConfig; release: ReleaseConfig}>,
+): Promise<TargetLabel[]>;
```

Comment:
Are these overloads really needed here? Seems like the implementation signature already allows for these? We should remove them for readability.

Reply:
The overloads are just in place so that we can provide the values in the `integration.spec.ts` tests.

Reply:
I don't follow how these all overloads combined, are different from just having a single signature like:

```ts
export async function getTargetLabels(
  api = GitClient.get().github,
  config = getConfig() as Partial<{github: GithubConfig; release: ReleaseConfig}>,
): Promise<TargetLabel[]> {}

// both still work; regardless of overloads
getTargetLabels();
getTargetLabels(githubApi, config);
```

Reply:
I though I had removed the overloads when I was able to use the default parameters.  Fixed.